packages/jumpstarter-driver-flashers/oci_bundles/aarch64-itb/build_fits.sh

```
@@ -4,7 +4,7 @@ cd "$(dirname "$0")"
 
 # run only in a container
 if [[ -z "$container" && ! -f /.dockerenv ]]; then
-    exec podman run --rm -it -v $(pwd):/host -w /host fedora:42 "$0" "$@"
+    exec podman run --rm -it -v $(pwd):/host:Z -w /host fedora:42 "$0" "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)